onebusaway-android/src/main/java/org/onebusaway/android/ui/NavigationDrawerFragment.java

```diff
@@ -289,10 +283,10 @@ private void setSelectedNavDrawerItem(int itemId) {
     }
 
     @Override
```

Comment:
@barbeau `onAttach(Activity activity)` is deprecated. Thats why I used `onAttach` with `Context`. This is the reference https://developer.android.com/reference/android/app/Fragment.html#onAttach(android.app.Activity). Do we still want to use `onAttach(Activity activity)`?